src/routing/index.js

```diff
@@ -27,6 +27,8 @@ const Routing = () => {
         component={ Class }
       />*/}
       <PrivateRoute exact path={PATHS.CLASS} component={CreateClass} />
+
+      <PrivateRoute exact path={PATHS.CLASSLIST} component={ClassList} />
```

Comment:
We can also remove this `<PrivateRoute exact path={PATHS.CLASSLIST} component={ClassList} />`.

Reply:
done